lib/features/pokemon_list_connector.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:flutter/material.dart';
+import 'package:pokedex/features/pokemon_list_page.dart';
```

Comment:
for this feature maybe you can create a new directory for this something like `pokemon_list` => `lib/features/pokemon_list`, so we can keep things organize for the long run 🙂

Then move all inside all your pokemon_list related classes 🙂